test/Geometry/GeometryTestAlg.cxx

```diff
@@ -2338,6 +2344,35 @@ namespace geo{
   }
 
   //......................................................................
+  unsigned int GeometryTestAlg::planeAlignmentTo
+    (geo::PlaneGeo const& plane, geo::Vector_t const& wireDir) const
+  {
+    /*
+     * Return value:
+     *  * 0: not aligned with anything
+     *  * 1: aligned to width
+     *  * 2: aligned to depth
+     *  * 3: aligned to wires
+     */
```

Comment:
This is probably overkill but you could encode the intended return by using appropriately named enums rather than ints that rely on complementary comments

Reply:
Yeah, I thought of that. It would be the right thing to do from the C++ standpoint.
From the other standpoint, what I really need is only whether it is `0` or not, at the point where the first version was returning a boolean value. I could not resist to make the code uselessly more generic.
All in all, I think I will revert to `bool`.